src/helpers/Initialization.tsx

```diff
@@ -0,0 +1,24 @@
+﻿import MyDataHelps, { EventName } from '@careevolution/mydatahelps-js';
+import { DependencyList, useEffect } from 'react';
+
+const debounce = require('lodash.debounce');
```

Comment:
this is breaking downstream apps that use vite. I see the following when starting up the MyDatahelpsStarterKit:
<img width="847" alt="Screenshot 2023-06-26 at 3 30 38 PM" src="https://github.com/CareEvolution/MyDataHelpsUI/assets/32501270/895b83c4-c400-4687-9c44-18f00c7bc1e2">

Reply:
this looks to be a bug with vite: https://github.com/vitejs/vite/issues/3409 but I am not sure that has been published yet?

Reply:
Oh yeah, don't use require...I had to remove a bunch of require stuff out of here when moving to vite

Reply:
Taking a look now.

Reply:
https://github.com/CareEvolution/MyDataHelpsUI/pull/78